src/game_initialization/playcampaign.cpp

```diff
@@ -205,6 +206,7 @@ level_result::type campaign_controller::play_game()
 				}
 			}
 
+			gui2::switch_theme(prefs::get().gui2_theme());
```

Comment:
This could use a comment. If this is supposed to change the theme back when some scenario changed it then I don't think this is the right place.

Reply:
> This could use a comment.

Added.

> If this is supposed to change the theme back when some scenario changed it then I don't think this is the right place.

What would be a better place?

Reply:
The `campaign_controller` destructor perhaps.

Reply:
Moved to `campaign_controlle`r's destructor.